<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>Form</class>
    <widget class="QWidget" name="Form">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>672</width>
                <height>259</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Form</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
            <property name="leftMargin">
                <number>2</number>
            </property>
            <property name="topMargin">
                <number>2</number>
            </property>
            <property name="rightMargin">
                <number>2</number>
            </property>
            <property name="bottomMargin">
                <number>2</number>
            </property>
            <item>
                <layout class="QHBoxLayout" name="horizontalLayout">
                    <item>
                        <widget class="QLabel" name="node_label">
                            <property name="text">
                                <string>Node_Name</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QPushButton" name="pts_btn">
                            <property name="text">
                                <string>Pts</string>
                            </property>
                            <property name="checkable">
                                <bool>true</bool>
                            </property>
                            <property name="checked">
                                <bool>false</bool>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QPushButton" name="vtx_btn">
                            <property name="text">
                                <string>Vtx</string>
                            </property>
                            <property name="checkable">
                                <bool>true</bool>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QPushButton" name="prim_btn">
                            <property name="text">
                                <string>Prim</string>
                            </property>
                            <property name="checkable">
                                <bool>true</bool>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QPushButton" name="detail_btn">
                            <property name="text">
                                <string>Detail</string>
                            </property>
                            <property name="checkable">
                                <bool>true</bool>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QLabel" name="label_2">
                            <property name="text">
                                <string>Group:</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QLineEdit" name="search_line"/>
                    </item>
                    <item>
                        <widget class="QPushButton" name="refresh_btn">
                            <property name="text">
                                <string>Refresh</string>
                            </property>
                        </widget>
                    </item>
                </layout>
            </item>
            <item>
                <widget class="QTableWidget" name="info_table">
                    <property name="minimumSize">
                        <size>
                            <width>80</width>
                            <height>60</height>
                        </size>
                    </property>
                </widget>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections>
        <connection>
            <sender>detail_btn</sender>
            <signal>clicked()</signal>
            <receiver>prim_btn</receiver>
            <slot>toggle()</slot>
            <hints>
                <hint type="sourcelabel">
                    <x>343</x>
                    <y>14</y>
                </hint>
                <hint type="destinationlabel">
                    <x>262</x>
                    <y>14</y>
                </hint>
            </hints>
        </connection>
    </connections>
</ui>
